server/pages/index.jade

```diff
@@ -55,6 +55,9 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 			else
 				link(rel='stylesheet', href=urls['style.css'])
 
+		if oauth
+			link(rel='stylesheet', href='/calypso/css/' + oauth + '.css')
```

Comment:
Ben mentioned in Trello that we will have to customize the signup for each OAuth client, so these stylesheets might ended up being larger than expected.